src/modules/NewsList/test.tsx

```diff
@@ -148,6 +163,76 @@ describe('NewsList', () => {
         screen.getByText('Hello world');
     });
 
+    test('go back to list if company changes via message bus', async () => {
+        jest.spyOn(hooks, 'useCompanyResolver').mockReturnValue(() =>
```

Comment:
Any reason you didn't just add another check in the `executeQuery` mock below to see if it's the resolver query and return this like we do for other queries?